runtime/compiler/codegen/J9TreeEvaluator.cpp

```diff
@@ -2204,10 +2204,10 @@ void J9::TreeEvaluator::preEvaluateEscapingNodesForSpineCheck(TR::Node *root, TR
    }
 
 
-///////////////////////////////////////////////////////////////////////////////////////
+
 // resolveCHKEvaluator - Resolve check a static, field or method. child 1 is reference
 //   to be resolved. Symbolref indicates failure action/destination
-///////////////////////////////////////////////////////////////////////////////////////
+
```

Comment:
The lines that contain only slashes won't be treated by Doxygen as comments that it processes.  May I ask you to put these two lines back in?